code/modules/wiremod/integrated_circuit.dm

```diff
@@ -148,6 +153,15 @@
 	if(shell)
 		to_add.register_shell(shell)
 
+/**
+ * Adds a component to the circuitboard through a manual action.
+ */
+/obj/item/integrated_circuit/proc/add_component_manually(obj/item/circuit_component/to_add, mob/living/user)
+	if (SEND_SIGNAL(src, COMSIG_CIRCUIT_ADD_COMPONENT, to_add, user) & COMPONENT_CANCEL_ADD_COMPONENT)
```

Comment:
I think you were supposed to rename this to COMSIG_CIRCUIT_ADD_COMPONENT_MANUALLY

Reply:
```suggestion
	if (SEND_SIGNAL(src, COMSIG_CIRCUIT_ADD_COMPONENT_MANUALLY, to_add, user) & COMPONENT_CANCEL_ADD_COMPONENT)
```